boards/BMS/Src/App/App_TractiveSystem.c

```diff
@@ -1,8 +1,13 @@
 #include <stdlib.h>
 #include <assert.h>
+#include <stdbool.h>
+#include "App_TractiveSystem.h"
 
 #define HIGH_RES_MAX_CURRENT_READING (50.0f)
 
+#define MAX_TS_CHARGE_CURRENT_AMPS (23.6f)
```

Comment:
Please confirm that I interpreted these limits correctly based on [this](https://ubcformulaelectric.atlassian.net/wiki/spaces/UFE/pages/12353537/BMS+Faults+Warnings#What-are-faults-%26-warnings%3F)

Reply:
multiply by 3. we have 3 cells in parallel in the pack and current sensing is at the pack level

Reply:
Gotcha. I'll update that